yt/frontends/ramses/io_utils.pyx

```diff
@@ -63,21 +77,24 @@ def read_amr(FortranFile f, dict headers,
 
             if ng == 0:
                 continue
-            # Skip grid index, 'next' and 'prev' arrays (they are used
-            # to build the linked list in RAMSES)
-            f.skip(3)
+            # Skip grid index, 'next' and 'prev' arrays
+            record_len = (ng * INT32_SIZE + INT64_SIZE)
+            f.seek(record_len * 3, 1)
```

Comment:
I have a *slight* preference for using `SEEK_CUR` but honestly it's not a big deal.

Reply:
Done!